### ISSUE TITLE
'NoneType' object is not subscriptable error

### ISSUE THREAD

**Issue by hamman33** [hardikvasa/google-images-download#373]
Item no.: 1 --> Item name = weller_12 Bourbon whiskey
Evaluating...
Starting Download...
'NoneType' object is not subscriptable

Now receiving this error for each download I try

**Comment by dtozik** [comment]
in google-images-download.py:

`if info is None:
            info = data[11]`

change 11 to 22

**Comment by livan3li** [comment]
> in google-images-download.py:
> 
> `if info is None: info = data[11]`
> 
> change 11 to 22

thank you very much! I love you.
i've been searching for a whole day to figure it out what's the problem. when i fix the problem i get a new error and i fix that, get another error. At the end of i could solve the problem by your help. As i see nobody take cares the repo for bug fixes..

**Comment by Dries-Fierens** [comment]
Are you able to get images? I'm also stuck with

> > in google-images-download.py:
> > `if info is None: info = data[11]`
> > change 11 to 22
> 
> thank you very much! I love you. i've been searching for a whole day to figure it out what's the problem. when i fix the problem i get a new error and i fix that, get another error. At the end of i could solve the problem by your help. As i see nobody take cares the repo for bug fixes..

Are you able to get images? I'm also stuck with this error.

**Comment by livan3li** [comment]
> Are you able to get images? I'm also stuck with
> 
> > > in google-images-download.py:
> > > `if info is None: info = data[11]`
> > > change 11 to 22
> > 
> > 
> > thank you very much! I love you. i've been searching for a whole day to figure it out what's the problem. when i fix the problem i get a new error and i fix that, get another error. At the end of i could solve the problem by your help. As i see nobody take cares the repo for bug fixes..
> 
> Are you able to get images? I'm also stuck with this error.

Yes, i've just downloaded 250 images. Check this link out [here ](https://github.com/hardikvasa/google-images-download/issues/371) i explained here how i solved.

**Comment by Dries-Fierens** [comment]
thank you it is finally working now as intended

**Comment by websonder** [comment]
its stop again, is there any solution so i can run it again?

**Comment by nnyj** [comment]
in google-images-download.py line 405:

`info = data[9]`

change 9 to 23

**Comment by marianoig4** [comment]
Same problem here, I had changed 
- 11 to 22 and 23 in 407 line
- 9 to 23 in 405 line
And stil not working for me.....

**Comment by dtozik** [comment]
def lookupJsonKey(self, object, s):
        def lookupImpl(object, s):
            isDict = False
            if isinstance(object, dict):
                isDict = True
            elif not isinstance(object, list):
                return None
            for val in object:
                if isDict:
                    if val == s:
                        return object[val]
                        
                obj = object[val] if isDict else val
                ret = lookupImpl(obj, s)
                if ret != None:
                    return ret
            return None
        return lookupImpl(object, s)

    # Format the object in readable format
    def format_object(self, object):
        data = object[1]
        main = data[3]
        info = self.lookupJsonKey(data, '2003')
        formatted_object = {}
        try:
            formatted_object['image_height'] = main[2]
            formatted_object['image_width'] = main[1]
            formatted_object['image_link'] = main[0]
            formatted_object['image_format'] = main[0][-1 * (len(main[0]) - main[0].rfind(".") - 1):]
            formatted_object['image_description'] = info[3]
            formatted_object['image_host'] = info[17]
            formatted_object['image_source'] = info[2]
            formatted_object['image_thumbnail_url'] = data[2][0]
        except Exception as e:
            print(e)
            return None
        return formatted_object

find a function 'def format_object(self, object):' and put that code patch there.

**Comment by wkrkd273** [comment]
its stop again, is there any solution so i can run it again?

**Comment by nnyj** [comment]
in google-images-download.py line 407:

`info = data[22]`

change 22 to 25

**Comment by jocelin110** [comment]
> in google-images-download.py line 407:
> 
> `info = data[22]`
> 
> change 22 to 25

I just can't help loving you, guy!!! You are the sweet angel who made my night!!

**Comment by Ali-NAIRDC** [comment]
nothing works for me i changed the values and still got the same results. My chrome version is 123.xxx.xxx.xx
